core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -710,6 +707,7 @@ public PropServerConfiguration(
             this.telemetryEnabled = getBoolean(properties, env, "telemetry.enabled", true);
             this.telemetryDisableCompletely = getBoolean(properties, env, "telemetry.disable.completely", false);
             this.telemetryQueueCapacity = Numbers.ceilPow2(getInt(properties, env, "telemetry.queue.capacity", 512));
+            this.o3PartitionPurgeListCapacity = getInt(properties, env, "cairo.o3.partition.purge.list.initial.capacity", 1);
```

Comment:
The default value used in `DefaultCairoConfiguration` is 64, not 1, so we need to sync the values.

Reply:
Default value is used in tests I think it's fine to be different from default value in server.conf

Reply:
Not just in tests, but in embedded mode. Many embedded API users use this class, so we should keep it in sync with the defaults from the props class. Tests should override the value of they need it.